scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -5,58 +5,20 @@ set -e
 # show commands
 set -x
 
-ARCH=$(uname -m)
-export CI="openshift"
-if [ "${ARCH}" == "s390x" ]; then
-    make configure-installer-tests-cluster-s390x
-elif [ "${ARCH}" == "ppc64le" ]; then
-    make configure-installer-tests-cluster-ppc64le
+export AMQP_URI="amqp://mquser:password123@35.224.22.100/"
```

Comment:
password directly exposed in an open repo?

Reply:
At the moment. That is where secrets come into play